webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -273,28 +299,35 @@ export default {
 }
 </script>
 
-<style lang="scss" scoped>
-.smallTag {
-  width: 100%;
-  position: relative;
-  left: 90%;
-}
-.post-title {
-  margin-top: $space-x-small;
-  margin-bottom: $space-xx-small;
+<style lang="scss">
+.contribution-form {
+  .ds-card-image.--blur-image img {
+    filter: blur(32px);
+  }
 
-  input {
-    border: 0;
-    font-size: $font-size-x-large;
-    font-weight: bold;
-    padding-left: 0;
-    padding-right: 0;
+  .blur-toggle {
+    text-align: right;
+
+    > .link {
+      display: block;
+    }
   }
-}
 
-.contribution-form {
   .ds-chip {
     cursor: default;
   }
+
+  .post-title {
+    margin-top: $space-x-small;
+    margin-bottom: $space-xx-small;
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-03T17:19:51Z" title="Tuesday, December 3rd 2019, 6:19:51 pm +01:00">Dec 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -183,6 +239,28 @@ export default {
     }),
   },
   methods: {
+    unBlur() {
+      if (this.checkedBlur) {
+        this.checkedChange()
+      }
+    },
+    checkedChange() {
+      if (this.$el) {
+        this.elem = this.$el.querySelector('img')
+      } else {
+      }
+      if (this.checkedBlur) {
+        this.elem.classList.remove('img-blur-in')
+        this.checkedBlur = false
+        this.form.checkbox = false
+      } else {
+        if (this.elem != null) {
+          this.elem.classList.add('img-blur-in')
+        }
+        this.checkedBlur = true
+        this.form.checkbox = true
+      }
+    },
```


I think we can improve this! there is an empty else after `line 250`... 
there are nested `if` inside `else` clauses, which could be refactored to use `else if` statements, no?
Also, it's a bit complicated to follow what's going on here... what are you trying to achieve in `checkedChange`?
I see it is called when `unBlur` is called, and when the component is `mounted`